cogs/helpermessage.py

```diff
@@ -0,0 +1,144 @@
+import discord
+import toml
+from discord.ext import commands
+from utils.commands import available_subcommands
+from utils.pagination import paginated_embed_menus, PaginationView
+
+class Helpermessage(commands.Cog):
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.data = toml.load('config.toml')
+        self.subjects = self.data['subjects']
+        self.helpermessages = self.data['helpermessages']
```

Comment:
Read and write these with `db` rather than using the `config.toml`. Make sure you include the corresponding changes in lines 115–116 for writing new helpermessages to `db`.